cunumeric/_ufunc/ufunc.py

```diff
@@ -55,6 +55,44 @@
 Multiple GPUs, Multiple CPUs
 """
 
+_MULTIOUT_UNARY_DOCSTRING_TEMPLATE = """{}
+
+Parameters
+----------
+x : array_like
+    Only integer and boolean types are handled.
```

Comment:
I think both `modf` and `frexp` accept floating point numbers as well.

Reply:
Should be addressed now